src/python/Core/GUI.py

```diff
@@ -562,6 +571,26 @@ def workspace(self, *args, **kwargs):
       return self.start(workspace = self._workspace(args[0]).name)
     else:
       return self.start(workspace = self.workspaces[0].name)
+# -----------------------------------------------------------------
+  @expose
+  @tools.params()
+  def jsrootfairy(self, *args, **kwargs):
+    try:
+      if len(args) >= 1:
+          for s in self.sources:
+            if getattr(s, 'jsonhook', None) == args[0]:
+              kwargs['jsroot'] = 'true'
+              data = s.getJson(*args[1:], **kwargs)
+              return data
+    except Exception, e:
+      o = StringIO()
+      traceback.print_exc(file=o)
+      log("WARNING: unable to produce JSROOT json: "
+          + (str(e) + "\n" + o.getvalue()).replace("\n", " ~~ "),
+          severity=logging.WARNING)
+
+    self._noResponseCaching()
+    return str(e);
```

Comment:
It is the same one as in jsonfairy. Exception blocks are the same in both functions.